src/entities/offerings.ts

```diff
@@ -179,6 +179,11 @@ export interface SubscriptionOption extends PurchaseOption {
    * The trial information for this subscription option if available.
    */
   readonly trial: PricingPhase | null;
+
+  /**
+   * The introductory price period for this subscription option if available.
+   */
+  readonly introPrice: PricingPhase | null;
```

Comment:
Not for this PR! 

This is good for now, But I’ve been thinking it would be a good idea to make the base price + subs duration + trial duration + intro price and duration easier to access than in the SubscriptionOption. We could potentially create nullable copies of this data at the StoreProduct level (except the base price, which should be non nullable I think), as we do for some of this data in the [Android SDK](https://github.com/RevenueCat/purchases-android/blob/178691123e9638616f5eff6f17bbb1cf46f2808e/purchases/src/main/kotlin/com/revenuecat/purchases/models/StoreProduct.kt#L35).

Having said that, I think this can come as future improvements and doesn’t belong to this PR.